### DISCUSSION TITLE
VSCode Web version compatibility

### DISCUSSION THREAD

**Discussion by Jeffrey04** [vscode-neovim/vscode-neovim#1464] [Q&A]
Just out of curiosity, is it possible to get this extension working in the web UI through a web browser? For instance, when editing directly through github `.` / gitpod web / `code tunnel` command ([remote tunnels](https://code.visualstudio.com/docs/remote/tunnels))? I am moving some of my coding work to a samsung's android tablet with keyboard cover, almost everything works out of the box, except neovim extension.

**Comment by xiyaowong** [comment]
If you can use the VS Code Terminal feature, then you can use this extension.

**Comment by xiyaowong** [comment]
github.dev
![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/766cb5dd-1477-4795-8811-5159fd5ae8f7)

vscode.dev
![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/f8bb2979-796e-44a6-a51f-373e7d6734dc)
